test/e2e/netman/pingpong/pingpong.go

```diff
@@ -154,44 +156,23 @@ type Edge struct {
 	To   uint64
 }
 
-func (e Edge) Equals(other Edge) bool {
-	return e.From == other.From && e.To == other.To || e.From == other.To && e.To == other.From
-}
-
 // edges returns a deterministic map of unique edges between chains.
-func (a *XDapp) edges() []Edge {
-	var all []Edge
-	for fromChainID := range a.contracts {
-		for toChainID := range a.contracts {
-			if fromChainID == toChainID {
-				continue
-			}
-
-			all = append(all, Edge{From: fromChainID, To: toChainID})
-		}
+func (a *XDapp) setEdges() {
+	var resp []Edge
+	arr := []Contract{}
```

Comment:
Please, change that to:
```suggestion
	var arr []Contract
```

https://go.dev/wiki/CodeReviewComments#declaring-empty-slices

Reply:
I'll create another PR for this tomorrow